scripts/release/make-release.py

```diff
@@ -7,9 +7,7 @@
 from tickets import check_tickets
 
 release_tag_pattern = re.compile(r"^v\d\.\d\.\d(-RC\d)?$")
-# Enable this in development to avoid pushing
-dry_run = True
-
+dry_run = False
```

Comment:
Can you make this a command line option?  We already use docopt in `src/generate_passwords.py` so that should make this not a big faff

Reply:
Good idea, done.